### ISSUE TITLE
Shelly 1 Mini

### ISSUE THREAD

**Issue by EvgenyZ89** [alexryd/homebridge-shelly-ng#131]
Hello everyone! Tell me how I can forward a new device to Homebridge that is not on the list, namely Shelly 1 Mini?

![IMG_7666](https://github.com/user-attachments/assets/5a1a6782-44bd-441f-99c0-24112c03b5c9)

**Comment by BlackWidooW** [comment]
Did you find a solution?

**Comment by EvgenyZ89** [comment]
> Did you find a solution?

yes, i start use this plugin - https://github.com/cubi1337/homebridge-shelly-ds9#readme

work much better

**Comment by BartmanEH** [comment]
> > Did you find a solution?
> 
> yes, i start use this plugin - https://github.com/cubi1337/homebridge-shelly-ds9#readme
> 
> work much better

it doesn't show support for _Shelly 1 Mini Gen3_ but it is confirmed to work?
